src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login was called. Params: login = {} ", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.debug("Method login worked successfully. Params: login = {}", login);
         return user;
     }
 
     private User findByLogin(String login) {
+        logger.debug("Method findByLogin was called. Params: {}", login);
         User user = new User(login, "1234");
```

Comment:
The password is hardcoded as '1234' in the `findByLogin` method. This is not secure and should be replaced with a proper user authentication mechanism. Consider retrieving the password from a secure source or database.